src/Parser/Mage/Fire/Normalizers/KaelthasUltimateAbility.js

```diff
@@ -0,0 +1,37 @@
+import SPELLS from 'common/SPELLS';
+
+import EventsNormalizer from 'Parser/Core/EventsNormalizer';
+
+class KaelthasUltimateAbility extends EventsNormalizer {
+  /**
+   * @param {Array} events
+   * @returns {Array}
+   */
+  normalize(events) {
```

Comment:
Please include in the documentation comment above what the issue is with unmodified event ordering, and what this Normalizer does to remedy the issue.

It's often difficult to discern exactly what a Normalizer is doing (and why), which is why I'm insisting on commenting in this case.